### PR TITLE
(brazilian) improve translation quality

### PR DESCRIPTION
**Pull request by Tarkoviski**
Heyo!

This pull request contains some updates to improve the overall quality of the Brazilian translation. The changes address various issues such as incorrect translations, grammatical errors, and inconsistent phrasing.

Files changed:
- rich_presence (full review)
- store_page (full review)
- community/
  - eula (full review)
  - inventory_service (_partial_ review)
  - points_shop (full review)
  - steam_input (full review)
  - workshop (full review)
- resource/
  - chat_brazilian.txt (full review)
  - reactivedrop_brazilian.txt (_partial_ review, mostly to match the changes made in the other files)
  - basemodui_brazilian.txt (same as above)

I plan to take a look at the other files and make any necessary updates as soon as I get a chance. Some of these translations have subpar quality and need substantial improvement.

Please let me know if you have any questions or concerns about these changes.

Thanks!

### CODE REVIEW

File: community/steam_input/steam_input_brazilian.vdf
Comment (BenLubar) [inline community/steam_input/steam_input_brazilian.vdf:12]: Marines have three equipment slots: primary, secondary, and extra. This button activates the extra slot.

**Review by FurtadoPires — CHANGES_REQUESTED** [review]
First of all, I would like to thank you for reviewing my one-man job translation :)

My considerations:
eula_brazilian.txt
-Line 3: I think the phrase "Não fazer isso pode resultar" is too informal to a legal document, I would suggest to change to "O não cumprimento" or keep it as it is.

inventory_service_tags_brazilian.vdf
-Line 6: I suggest to keep this line unchanged, maintaining it similar to the english version

item-schema-donations.json
-Suggest to keep the original phrase "As doações não são necessárias para acessar nenhum conteúdo e não afetam a jogabilidade." due to being more similar to the original english

line 73: Please check if the special key "º" is displaying correctly before the change. Usually I avoid using special characters due to some unexpected bugs

lines 111, 118, 125: While "Parasita do Chapéu" could be fitting name for that item, I suggest to keep the original translation due to this being a "joke" that Ben did there. If you check the other translations you can see that everybody kept the original name and description :)

mappingcontest2023_brazilian.txt
-Line 1: Could be "Enfrente o Chefão" but you can keep your translation if you want

reactivedrop_brazilian.txt
-lines 132, 212, 214: Suggest to keep "Permutador" (it's a machine that exchange heat)

-Line 200: I've kept the original "snow cats" name since it appear to be a fictional "brand" of machines. Also the part "incluindo mas não limitado a" sounds more like something a company contract would say :)

-Line 579: Apparently I forgot to change the IAF there, if you could do me a favor and correct that for me :)

-Line 589: There's a typo in your alteration "Exluir"

563560_loc_brazilian.vdf
-Lines 9 to 16: Not sure why to change there since apparently is the format to the original english file, just make sure it displays ok in the game if you want to change that

Other than that, I don't see any problems with the changes

File: community/steam_input/steam_input_brazilian.vdf
Comment (FurtadoPires) [reply community/steam_input/steam_input_brazilian.vdf:12]: @BenLubar which word could be used as a synonym to "off-hand" if you had to change it?

**Comment by FurtadoPires** [comment]
Very well then.

My only "nitpick" will be the translation of the hat, the point is to keep the original joke of the name and description (like everybody else did), fortunately this isn't a lost in translation case and the name makes sense if you read the original english description.

I won't hold this PR just for that (in case you want to maintain this change), but I might revert it in the future. Other than that I'm ok with the other changes since it doesn't affect the compression and you guarantee will not generate any visual glitches.

Thank you.

File: community/steam_input/steam_input_brazilian.vdf
Comment (DukeDukeEm) [reply community/steam_input/steam_input_brazilian.vdf:12]: I had a similiar situation in my german translation. At some point I settled on one word which translates back well too.
Maybe you can find an inspiration for these back-translations into english:
https://dict.leo.org/german-english/Zusatzausr%C3%BCstung

Comment (BenLubar) [reply community/steam_input/steam_input_brazilian.vdf:12]: > How do you feel about using "item adicional" ("additional item") as an alternative for "offhand item"?

That sounds good to me.

**Comment by BenLubar** [comment]
By the way, can I have a link to your Steam profile? You've more than earned the Translator medal.

File: resource/reactivedrop_brazilian.txt
Comment (BenLubar) [reply resource/reactivedrop_brazilian.txt:909]: Check the beta branch. For me, this is how it looks with the current English translation:

image

Comment (BenLubar) [reply resource/reactivedrop_brazilian.txt:909]: Fixed the bug in https://github.com/ReactiveDrop/reactivedrop_public_src/commit/3b60142f80e5e1ccdeb3dde317f1e25cd46c2dd5 (the single percent sign is the correct variant)

**Comment by BenLubar** [comment]
@FurtadoPires is this okay to merge in its current state?

**Review by FurtadoPires — APPROVED** [review]

**Comment by FurtadoPires** [comment]
@BenLubar All ok for me

**Comment by BenLubar** [comment]
@Tarkoviski @FurtadoPires one last question before I merge: which version of the Trick Hat translation are we using?
image

**Comment by BenLubar** [comment]
image

ok, problem with the workshop special area title: it's too long for the box

**Comment by BenLubar** [comment]
> Also, I didn't fix the issue in resource/reactivedrop_brazilian.txt#L909 (`%%` → `%`). I was planning to address it alongside any other changes requested by Furtado. So sorry about that :/

I'll fix that in my next commit.

**Pull request merged**
